mergJSON/mergJSONLibrary.livecodescript

```diff
@@ -113,14 +117,18 @@ translated to JSON objects.
 */
 
 function ArrayToJSON pArray,pForceRootType,pPretty
-   repeat for each key tKey in pArray
-      if pArray[tKey] is an array then
-         put "}"&ArrayToJSON(pArray[tKey],pForceRootType) into pArray[tKey]
-      else if char 1 of pArray[tKey] = "}" then
-         put "}}" before pArray[tKey]
-      else
-         put textEncode(pArray[tKey],"UTF8") into pArray[tKey]
-      end if
-   end repeat
-   return(mergJSONEncode("pArray",pForceRootType,pPretty))
+   if pArray is not empty then
```

Comment:
@mwieder I'm fairly sure this is wrong and should be `if pArray is an array then`